README.md

```diff
@@ -28,36 +31,15 @@ The name *Slint* is derived from our design goals:
     flexibility that only a native application can offer: Access full operating
     system APIs, utilize all CPU and GPU cores, connect to any peripheral.
 
-Visit [#MadeWithSlint](https://madewithslint.com) to view some of the projects
-using Slint. We invite you to use Slint and be part of its community.
-
-## Current Status
+Beyond the design goals, here’s what makes Slint stand out:
 
-Slint is in active development. The state of support for each platform is as
-follows:
+- **Design UI Independently**: Use a declarative language similar to separate your UI from business logic. Designers can work in parallel with developers.
+- **Tooling**: Including out editor integration and Live Preview & Editing to iterate quickly. We also have a Figma plugin.
```

Comment:
"Inclusion" doesn't seem like a term used commonly in the context of IDEs/editors. How about we put the verb in front that we really want the reader to catch?

```suggestion
- **Tooling**: Iterate quickly with our Live Preview & editor integrations. Integrate from Figma with the [Slint To Figma plugin](https://www.figma.com/community/plugin/1474418299182276871/figma-to-slint).
```